extensions/strategies/trust_distrust/strategy.js

```diff
@@ -0,0 +1,136 @@
+var z = require('zero-fill')
```

Comment:
Could you comment what exactly this strategy does at the top?  What you wrote in the description of this pr would be really awesome.